### ISSUE TITLE
Style/Documentation with only macros

### ISSUE THREAD

**Issue by marcandre** [rubocop-hq/rubocop#8788]
I'd like Style/Documentation to accept the following:

```ruby
module RuboCop
  module AST
    # Responsible for compatibility with main gem
    # @api private
    module RuboCopCompatibility
      # ...
    end

    extend RuboCopCompatibility  # => This triggers Style/Documentation: Missing top-level module documentation comment
  end
end
```

I think that (at least) macro calls like `extend` should not trigger it.

**Comment by carlos-ssh** [comment]
Great job guys!!!
